### ISSUE TITLE
Error - git rev-parse ambiguous

### ISSUE THREAD

**Issue by PyroSA** [tnesbitt210/interactive-smartlog-issues#11]
Installed version 0.1.35 to check it out as an alternative to Git Graph.
VS Code 1.89.1 under Windows 10.

Errors while loading commits.

```
Error with executing git rev-parse 'refs/heads/branch/name' from C:/code/repo: Error: Command failed: git rev-parse 'refs/heads/branch/name'
fatal: ambiguous argument ''refs/heads/branch/name'': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

Loading commits failure
```

**Comment by tnesbitt210** [comment]
I pushed an update - this should be fixed as of v0.1.36

**Comment by PyroSA** [comment]
Thanks @tnesbitt210 - It works in 0.1.36.

1 - Yes, works fine.
2 - These specific branches were unique, I couldn't spot any obvious issues, hence _this_ issue.
